lib/libtlspool.c

```diff
@@ -627,12 +649,14 @@ int tlspool_starttls (int cryptfd, starttls_t *tlsdata,
 				if ((plainfd < 0) && (cmd.pio_cmd == PIOC_PLAINTEXT_CONNECT_V2)) {
 					int soxx [2];
 					//TODO// Setup for TCP, UDP, SCTP
-					if (socketpair (AF_UNIX, SOCK_SEQPACKET, 0, soxx) == 0) {
```

Comment:
You must not modify the semantics for UNIX by moving away from SEQPACKET.  If you need to on Windows, make this dependent on __CYGWIN__ please.